hack/create.sh

```diff
@@ -25,6 +25,23 @@ if [[ -z "$AZURE_CLIENT_SECRET" ]]; then
     exit 1
 fi
 
+if [[ -z "$AZURE_REGION" ]]; then
```

Comment:
I know it's painful and unfriendly but I'd prefer to replace all this with

```
if [[ -z "$AZURE_REGION" ]]; then
    echo error: must set AZURE_REGION
    exit 1
fi
```

Fundamentally I want this script to die, not become a user friendly crutch.

Reply:
Doing the above may imply that a separate commit is needed to fix up our tests.  If so, please have a go - look in openshift/release.

Reply:
ok, I had this initially then thought of the build job... https://github.com/openshift/release/pull/1588